### DISCUSSION TITLE
conductor Spring boot Vulnerability CVE-2016-1000027		 Critical

### DISCUSSION THREAD

**Discussion by shekarpcs** [Netflix/conductor#3406] [General]
This is latest vulnerabilities on conductor 13 version. We are stuck in production. Anyway we can fix this and release small version

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/d115501/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/d115501/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>

</style>
</head>

<body link=blue vlink=purple>

56 |   | 47 | critical | 9.8 | fixed in 1.31 | CVE-2022-1471 |   | SnakeYaml\'s   Constructor() class does not restrict types which can be instantiated during   deserialization. Deserializing yaml content provided by an attacker can lead   to remote code execution. We recommend using SnakeYaml\'s SafeConsturctor   when parsing untrusted content to restrict deserialization. | CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H | https://nvd.nist.gov/vuln/detail/CVE-2022-1471 | image | org.yaml_snakeyaml | 1.30
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
57 |   | 47 | critical | 9.8 | fixed in 6.0.0 | CVE-2016-1000027 |   | Pivotal Spring Framework through   5.3.16 suffers from a potential remote code execution (RCE) issue if used for   Java deserialization of untrusted data. Depending on how the library is   implemented within a product, this issue may or not occur, and authentication   may be required. NOTE: the vendor\'s position is that untrusted data is not   an intended use case. The product\'s behavior will not be changed because   some users rely on deserialization of trusted data. | CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H | https://nvd.nist.gov/vuln/detail/CVE-2016-1000027 | image | spring-core_spring-core | 5.3.22

</body>

</html>

**Comment by sunchencc** [comment]
挂机  出去玩O(∩_∩)O
